src/components/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
```

Comment:
`useState` is not being used in this component so the import here can be deleted 
```
import React from "react";
```